src/catalog/table.rs

```diff
@@ -38,14 +39,31 @@ impl TableCatalog {
                 columns: BTreeMap::new(),
                 is_materialized_view,
                 next_column_id: 0,
+                primary_key_ids: Vec::new(),
             }),
         };
+        let mut pk_ids = vec![];
         for col_catalog in columns {
+            if col_catalog.is_primary() {
+                pk_ids.push(col_catalog.id());
+            }
             table_catalog.add_column(col_catalog).unwrap();
         }
+
+        table_catalog.set_primary_key_ids(&pk_ids);
```

Comment:
It seems that this logic still cannot tell the different between `primary key (a, b)` and `primary key (b, a)`? May fix this in later PRs.

Reply:
`primary_key_ids` is a vector， maybe we can tell the difference by index？